test/src/browser.spec.ts

```diff
@@ -64,7 +64,7 @@ describe('Browser specs', function () {
     });
   });
 
-  describe('Browser.isConnected', () => {
+  describe('Browser.connected', () => {
```

Comment:
Let's revert unrelated changes. Also https://pptr.dev/api/puppeteer.browser.isconnected there is some issue with the documentation for Browser.connected

Reply:
FYI It's sort a fixed on `next`.